README.md

```diff
@@ -28,7 +28,12 @@ esaba hosts your markdown docs on [esa.io](https://esa.io). Url like `/post/:pos
 - [Composer](https://getcomposer.org/)
 - [npm](https://www.npmjs.com/)
 
+Or
+
+- [Docker](https://www.docker.com/)
```

Comment:
Strictly speaking, this code requires not only docker, but also docker-compose.

At Linux platform, docker-compose is not contained in docker package. https://docs.docker.com/compose/install/

Reply:
fixed https://github.com/standfirm/esaba/pull/2/commits/b3851fb9dbf3acbb08caade0c50302f22c222880